sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -1,20 +1,14 @@
 # Azure.Search.Documents Code Generation
 
-Run `/sdk/search/generate.ps1` to generate code.
```

Comment:
Nit - can we replace with the dotnet command required or a link to where folks can find the latest from Contributing.md?